pages/docs/dragonite/config.mdx

```diff
@@ -89,20 +106,41 @@ bearer_token = "KOJI_SECRET"
 [tuning]
 #recycle_gmo_limit = 4900
 #recycle_encounter_limit = 9900
+#recycle_invasion_limit = 940
 #recycle_on_jail=false
 #minimum_account_reuse_hours = 169
-#location_delay = 0
-#fort_location_delay = 0
+#scout_age_limit = 30
+location_delay = 3000
+fort_location_delay = 3000
+quest_layer_swap = false
+test_shadow_banned = true
+
 ```
 
 | Key | Default | Description |
 | :-- | :-- | :-- |
 | `recycle_gmo_limit` | 4900 | (Advanced) Recommend keeping this value commented out. When the GMO limit is reached for a specific account it will be be rotated. |
```

Comment:
these values will gonna change by default in the future:
recycle_gmo_limit = 4990
recycle_encounter_limit = 9975
recycle_invasion_limit = 900